DiamondLightSource/python-artemis
# 554 tidy oav

Fixes DiamondLightSource/hyperion#554 

Link to dodal PR (if required): [#25](https://github.com/DiamondLightSource/dodal/pull/25)

### To test:
1. Check organisation/naming of oav code is sensible/coherent
2. Run tests with both artemis and dodal branches
